.github/workflows/publish.yml

```diff
@@ -40,8 +40,51 @@ jobs:
       - name: Install mdbook
         uses: ./.github/workflows/install-mdbook
 
+      - name: Install mdbook-pandoc
+        run: curl --proto '=https' --tlsv1.2 -LsSf https://github.com/max-heller/mdbook-pandoc/releases/download/v0.2.0/mdbook-pandoc-installer.sh | sh
+      - name: Install pandoc
+        run: |
+          curl -LsSf https://github.com/jgm/pandoc/releases/download/3.1.9/pandoc-3.1.9-linux-amd64.tar.gz | tar zxf -
+          echo "$PWD/pandoc-3.1.9/bin" >> $GITHUB_PATH
+      - name: Install mdbook-pandoc dependencies
+        run: sudo apt-get install -y librsvg2-bin
+      - name: Install TeX Live
+        uses: teatimeguest/setup-texlive-action@v3
```

Comment:
Would it be possible to simply this by simply installing Pandc and TeX Live using system packages?

Debian (and thus Ubuntu) has normally been good at including TeX packages — at least in my experience ~10 years ago where I always used the ones from the system instead of dealing with hand-installed stuff.

Reply:
On my Debian Unstable machine, this worked fine to get a working PDF:

```
sudo apt install pandoc texlive texlive-luatex librsvg2-bin
```

Reply:
This works well for TeX, but the repository for Ubuntu 22.04 (`ubuntu-latest` in Github Actions) includes a pretty old version of Pandoc that's missing extensions needed for optimal rendering

Reply:
> This works well for TeX, but the repository for Ubuntu 22.04 (`ubuntu-latest` in Github Actions) includes a pretty old version of Pandoc that's missing extensions needed for optimal rendering

Okay, fair enough! I'm glad that the installation instructions are much simpler now!